backend/src/ml_space_lambda/utils/common_functions.py

```diff
@@ -222,15 +222,15 @@ def generate_exception_response(e, status_code=400):
             },
             {
                 "Codes": ["AccessDeniedException"],
-                "FriendlyMessage": "An administrator or owner has restricted access to this resource. If you need access, please contact a system administrator or owner of the resource for assistance",
+                "FriendlyMessage": "An administrator or owner has restricted access to this resource or this account has hit its service limit for this resource. If you need access or to increase the limits, please contact a system administrator or owner of the resource for assistance",
             },
         ]
 
         for errorType in ERROR_DICT:
             if e.response["Error"]["Code"] in errorType["Codes"] and (
                 "MatchStrings" not in errorType or any(error in error_msg for error in errorType["MatchStrings"])
             ):
-                e = f"{errorType['FriendlyMessage']} Full message: {error_msg}"
+                e = f"{errorType['FriendlyMessage']}. Full message: {error_msg}"
```

Comment:
Are we sure all our error messages don’t have a period so this isn’t doubling up? I think it might be better to just change our error messages instead of inserting the period here.

Reply:
good idea, will do

Reply:
That or add some sort of conditional like

`errorType['FriendlyMessage'].endsWith(".") ? errorType['FriendlyMessage'] : errorType['FriendlyMessage'].`